### DISCUSSION TITLE
First impressions feedback

### DISCUSSION THREAD

**Discussion by manim-examples** [flxzt/rnote#955] [Ideas]
Here's some feedback from the first hour of using this app, which is very smooth and useful by the way. I mostly only use desktop software, and so this is coming from the Windows app.

I'll use emojis to mark my observations:

| Emoji  | Meaning |
| ------------- | ------------- |
| ℹ  | Would be useful to  |
| ☹  | Was confusing to find  |
| 🚩  | Would be a lot easier  |

So here's the list:

* ℹ show mouse cursor hand when hovering over icons/menus. Currently, it's a fixed arrow cursor everywhere (except when you hover over a value field to change font size or similar, then it's becomes the ibeam cursor). See the current look of it:

![cursor](https://github.com/flxzt/rnote/assets/145919594/605887d6-cafe-4711-b1b0-a79aaf87c3c8)

* ℹ higher contrast in dark colour scheme. See image above, where the button highlight is not that visible, at least on my lower brightness monitor. I have resorted to using light theme for now, it seems to look better than dark colour scheme for me.

* ℹ know what the default font is.

* ☹ what the fill colour is for. I thought maybe it's another stroke for the ink sice I couldn't easily find any shapes (circles, triangles ...) at first. Then I noticed they were inside the `Shape Builders` menu, which I guess makes sense. It could use a triangle, or even better, an n-gon generator, similar to the one in Figma. And some other shapes that are otherwise available in MS Paint. I like how you made the `Grid` work, quite unique I think.

* ☹ the settings menu. The more important buttons in the top bar are on the right side, even after changing the sidebar's position. There's the burger menu and the wrench icon, both of which have some of their own settings that can be configured. Perhaps I'm exaggerating with this one, but I like to have all file or settings related options on the top left, and in one place. Or have an option to convert the top bar to have names for menus instead of icons.

* 🚩 having a simple mouse pointer feature. I did find `Select One by One`, but that keeps on selecting elements. The proposed mouse pointer would be just as simple, except it would select only one element at once. Possibly more with holding down Shift. I'm aware this is going too much towards desktop, but is also really handy to have.
  * Especially when pasting in an image, because after successfully pasting, I have to select the image with then automatically selected `Selector` tool. This is good, but I guess the `Selector` tool can be any of its four subtools. I think consistancy here would be useful to have? Maybe an option to always toggle to the same `Selector` subtool after pasting an image? Or, instead of being redirected to one of the `Selector` tools upon pasting an image, you could redirect to the new mouse pointer tool that would be its own tool.
 
* 🚩 having universal shortcuts to move around, such as:
  * `Shift + Scroll Wheel` to move left and right.
  * <s>`Alt + Scroll Wheel` to move up and down.</s>
  * `Space + Move Mouse` to move around.
  * These are all available in Figma, and I assume in some other similar software too.

* ℹ exit out of the submenus by clicking away. Might be a bug; I notice that I cannot exit a submenu after focusing on a dropdown inside a subsubmenu. See provided GIF:

![focus](https://i.gyazo.com/63b50e6b5d155238dbc5feac30bdb774.gif)

* ℹ have mouse cursor as an option among other in `Regular Cursor` over in the `General Settings`.

* ℹ have the following `Ctrl + ...` shortcuts:
  * `A` select all elements in the current viewport.
  * `T` open a new tab.
  * `Shift + T` open the last closed tab.
  * Last two are both used in browsers and some text editors.

Other than that, no hate, I'm aware that most of these request are too desktop sided, but regardless of them eventually being implemented, I'm glad I found this app on alternativeto.net and later also on Reddit. Unfortunately, there doesn't seem to be anyone talking about it on YouTube just yet.

**Comment by Doublonmousse** [comment]
`ℹ exit out of the submenus by clicking away. Might be a bug`
Yes, this is a longstanding bug, see #206, waiting on upstream gtk4 to be fixed.

Some of the remarks were things already mentionned before (like the fill being hard to understand at first), but I'll stop there today

**Reply by Doublonmousse** [reply]
The default font is Cantarell, see [Gnome HIG](https://developer.gnome.org/hig/guidelines/typography.html)

**Reply by Doublonmousse** [reply]
`Space + Move Mouse` to move around : There is `Alt+ Drag` to move around already. You can also move around using `middle click + drag`
